### PR TITLE
New package: rsgain-3.4.0

### PR DESCRIPTION
**Pull request by dezifit**
rsgain (really simple gain) is a very heavily modified fork of [loudgain](https://github.com/Moonbase59/loudgain), and is licensed accordingly under the original 2 clause BSD license used by loudgain. 

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l (only compiled, not tested)

### CODE REVIEW

File: srcpkgs/rsgain/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'rsgain'
+pkgname=rsgain
+version=3.4
+revision=0
```
Comment (oreo639) [inline srcpkgs/rsgain/template:4]: This should be `revision=1`

File: srcpkgs/rsgain/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libebur128-devel taglib-devel ffmpeg-devel inih-devel fmt-devel"
+short_desc="ReplayGain 2.0 loudness normalizer"
+maintainer="dezifit@gmx.de"
+license="BSD-2-Clause"
+homepage="https://github.com/complexlogic/rsgain"
```
Comment (cinerea0) [inline srcpkgs/rsgain/template:11]:
```suggestion
homepage="https://github.com/complexlogic/rsgain"
changelog="https://raw.githubusercontent.com/complexlogic/rsgain/master/CHANGELOG"
```

File: srcpkgs/rsgain/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+distfiles="https://github.com/complexlogic/rsgain/archive/refs/tags/v3.4.tar.gz"
```
Comment (cinerea0) [inline srcpkgs/rsgain/template:12]:
```suggestion
distfiles="https://github.com/complexlogic/rsgain/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/rsgain/template [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'rsgain'
+pkgname=rsgain
+version=3.4
+revision=0
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libebur128-devel taglib-devel ffmpeg-devel inih-devel fmt-devel"
+short_desc="ReplayGain 2.0 loudness normalizer"
+maintainer="dezifit@gmx.de"
```
Comment (cinerea0) [inline srcpkgs/rsgain/template:9]:
```suggestion
maintainer="dezifit <dezifit@gmx.de>"
```

**Pull request closed without merging**